bitnami/node-exporter/Chart.yaml

```
@@ -28,4 +28,4 @@ maintainers:
 name: node-exporter
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/node-exporter
-version: 4.1.1
+version: 4.1.2
```

Because this PR adds a new feature, version should be bumped a minor instead of a patch:
```suggestion
version: 4.2.0
```